examples/overflow.ts

```diff
@@ -60,22 +60,6 @@ printTable({
   titleOptions: {bold: true},
 })
 
-// I would expect this for wrapping on align-center:
-//
-// Wrap (aligned center)
-// ┌───────┬─────────┬─────┬───────────────────────────────────────────────────────────────────────────┐
-// │  Id   │  Name   │ Age │                                Description                                │
-// ├───────┼─────────┼─────┼───────────────────────────────────────────────────────────────────────────┤
-// │ 36329 │  Alice  │ 20  │  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod  │
-// │       │         │     │            tempor incididunt ut labore et dolore magna aliqua.            │
-// ├───────┼─────────┼─────┼───────────────────────────────────────────────────────────────────────────┤
-// │ 49032 │   Bob   │ 21  │  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod  │
-// │       │         │     │            tempor incididunt ut labore et dolore magna aliqua.            │
-// ├───────┼─────────┼─────┼───────────────────────────────────────────────────────────────────────────┤
-// │ 51786 │ Charlie │ 22  │  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod  │
-// │       │         │     │            tempor incididunt ut labore et dolore magna aliqua.            │
-// └───────┴─────────┴─────┴───────────────────────────────────────────────────────────────────────────┘
-
 printTable({
```

Comment:
I am seeing the right edge of the rows being off by one space occasionally.
I am only seeing this on (aligned center) and (aligned left) when there are 3+ wraps

```
┌───────┬─────────┬─────┬─────────────────────────────────────────────────────────┐
│ Id    │ Name    │ Age │ Description                                             │
├───────┼─────────┼─────┼─────────────────────────────────────────────────────────┤
│       │         │     │ Lorem ipsum dolor sit amet, consectetur adipiscing     │
│ 36329 │ Alice   │ 20  │ elit. Sed do eiusmod tempor incididunt ut labore et    │
│       │         │     │ dolore magna aliqua.                                   │
├───────┼─────────┼─────┼─────────────────────────────────────────────────────────┤
│       │         │     │ Lorem ipsum dolor sit amet, consectetur adipiscing     │
│ 49032 │ Bob     │ 21  │ elit. Sed do eiusmod tempor incididunt ut labore et    │
│       │         │     │ dolore magna aliqua.                                   │
├───────┼─────────┼─────┼─────────────────────────────────────────────────────────┤
│       │         │     │ Lorem ipsum dolor sit amet, consectetur adipiscing     │
│ 51786 │ Charlie │ 22  │ elit. Sed do eiusmod tempor incididunt ut labore et    │
│       │         │     │ dolore magna aliqua.                                   │
└───────┴─────────┴─────┴─────────────────────────────────────────────────────────┘
```

Reply:
This looks good now!